uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-leave.controller.spec.js

```diff
@@ -170,6 +170,21 @@ define([
             expect(controller.canCalculateChange()).toBe(true);
           });
         });
+
+        describe('when unit changed to hours by changing an absence type', function () {
```

Comment:
`when the unit changes to hours because the absence type changed`

Reply:
Done.